packages/babel-plugin-transform-typescript/test/fixtures/imports/elide-injected/plugin.js

```diff
@@ -17,8 +17,8 @@ module.exports = function({ types: t }) {
           // with this crawl, babel-plugin-transform-typescript
           // used to elide the import we just added
           // https://github.com/babel/babel/issues/7592
-          path.scope.crawl();
```

Comment:
Shouldn't we keep both `crawl`s in? Without this one, the test no longer ensures that a plugin doing this works correctly and it could regress to eliding injected but not yet referenced imports